src/main/java/uk/gov/ons/ctp/response/action/service/ActionService.java

```diff
@@ -40,21 +50,44 @@
   private static final Logger log = LoggerFactory.getLogger(ActionService.class);
 
   private static final int TRANSACTION_TIMEOUT = 30;
+  private static final String ENABLED = "ENABLED";
 
-  @Autowired private ActionRepository actionRepo;
+  private static final String NO_RESPONDENTS_FOR_SURVEY = "No respondents found for survey";
 
-  @Autowired private ActionCaseRepository actionCaseRepository;
+  private ActionRepository actionRepo;
+  private ActionCaseRepository actionCaseRepo;
+  private ActionPlanRepository actionPlanRepository;
+  private ActionPlanJobRepository actionPlanJobRepository;
+  private ActionRuleRepository actionRuleRepo;
+  private ActionTypeRepository actionTypeRepo;
 
-  @Autowired private ActionTypeRepository actionTypeRepo;
+  private CollectionExerciseClientService collectionExerciseClientService;
+  private PartySvcClientService partySvcClientService;
```

Comment:
final

Reply:
Making these final was having strange effects on the tests. It seemed a bit challenging to fix the tests around it so leaving the final out for now